padertorch/train/trainer.py

```diff
@@ -272,6 +272,8 @@ def train(
                 # TypeError: object of type '...' has no len()
                 max_it_len = None
             hooks.append(ProgressBarHook(self._stop_trigger, max_it_len))
+            # set_last updates the iteration counter in case of resume
+            hooks[-1].set_last(self.iteration, self.epoch)
```

Comment:
Can you make the `set_last` call explicit? (i.e. `progress_bar_hook.set_last(...)`)

Reply:
Done